xslt/xml-to-html-statistics.xslt

```diff
@@ -32,15 +32,6 @@ Modified by Stephen Moon for DCO
 		xmlns:fn="http://www.w3.org/2005/xpath-functions"
 		exclude-result-prefixes="res xsl">
 
-  <xsl:template name="header">
-    <div>
-      <h2>Header</h2>
-      <xsl:for-each select="res:head/res:link"> 
-	<p>Link to <xsl:value-of select="@href"/></p>
-      </xsl:for-each>
-    </div>
-  </xsl:template>
-
```

Comment:
also below in the vb-result template there's a div tag that needs to be removed and its ending tag. We don't need that either. We can always add that if we'd like to.